src/trigger/text_area.cpp

```diff
@@ -88,6 +96,7 @@ TextArea::event(Player& player, EventType type)
         m_current_text = 0;
         m_status = Status::FADING_IN;
         m_timer.start(m_fade_delay);
+        text_object.set_anchor_point(string_to_anchor_point(anchor_points[m_anchor]));
         text_object.set_text(m_items[m_current_text]);
```

Comment:
With the two edits above, this line becomes simply `text_object.set_anchor_point(m_anchor);`